### PR TITLE
[#669] Handle one-way commands in the HTTP adapter.

### PR DESCRIPTION
**Pull request by sysexcontrol**
Contained is the implementation and documentation of one-way commands for the HTTP adapter.
Missing in this PR (will follow in next step) are integration tests and the extension of the example app.

Signed-off-by: Karsten Frank <Karsten.Frank@bosch-si.com>

### CODE REVIEW

File: site/diagrams/sequenceDiagrams/command_control_concept_http.txt [hunk 1]
```diff
@@ -7,7 +7,8 @@ activate HTTP Adapter
 HTTP Adapter->Application: 2. transfer("event/TENANT","4711","ttd=30")
 activate Application
 Application->HTTP Adapter: 3. transfer("control/TENANT/4711","light","ON")
-HTTP Adapter->Device: 4. HTTP response ("200", "hono-cmd=light, hono-cmd-req-id=${reqId})", "ON")
+HTTP Adapter->Application: disposition("accepted")
+HTTP Adapter->Device: 4. HTTP response (("200", "hono-cmd=light", "hono-cmd-req-id=${reqId}"), "ON")
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/command_control_concept_http.txt]: it is actually the other way around, first we send the response then we accept the transfer

File: site/diagrams/sequenceDiagrams/command_control_concept_one_way_http.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+participant HTTP Adapter
+participant Application
+
+Device->HTTP Adapter: 1. HTTP request ("POST", "/event", "hono-ttd=30")
+activate HTTP Adapter
+HTTP Adapter->Application: 2. transfer("event/TENANT","4711","ttd=30")
+activate Application
+Application->HTTP Adapter: 3. transfer("control/TENANT/4711","light","ON")
+HTTP Adapter->Application: disposition("accepted")
+HTTP Adapter->Device: 4. HTTP response (("200", "hono-cmd=light"), "ON")
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/command_control_concept_one_way_http.txt]: iagain, t is actually the other way around, first we send the response then we accept the transfer

File: site/diagrams/sequenceDiagrams/command_control_concept_http.txt (hunk 1 above)
Comment (sysexcontrol) [reply site/diagrams/sequenceDiagrams/command_control_concept_http.txt]: Oh yes, you are correct! Will change this...

**Comment by sysexcontrol** [comment]
@sophokles73 : made the changes.

**Pull request merged**